uk.co.compucorp.civicrm.hrcore/hrcore.php

```diff
@@ -280,16 +281,22 @@ function hrcore_civicrm_pageRun($page) {
  * Implements hook_civicrm_navigationMenu().
  *
  * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_navigationMenu/
+ *
+ * @param array $menu
+ *   The full navigation menu structure
  */
-function hrcore_civicrm_navigationMenu(&$params) {
-  _hrcore_renameMenuLabel($params, 'Contacts', 'Staff');
-  _hrcore_renameMenuLabel($params, 'Administer', 'Configure');
-
-  _hrcore_coreMenuChanges($params);
-  _hrcore_createHelpMenu($params);
-  _hrcore_createDeveloperMenu($params);
-  _hrcore_setDynamicMenuIcons($params);
-  _hrcore_createSelfServicePortalMenu($params);
+function hrcore_civicrm_navigationMenu(&$menu) {
+  _hrcore_renameMenuLabel($menu, 'Contacts', 'Staff');
+  _hrcore_renameMenuLabel($menu, 'Administer', 'Configure');
+
+  _hrcore_coreMenuChanges($menu);
+  _hrcore_createHelpMenu($menu);
+  _hrcore_createDeveloperMenu($menu);
+  _hrcore_setDynamicMenuIcons($menu);
+  _hrcore_createSelfServicePortalMenu($menu);
```

Comment:
Any plans to also create modifiers for the existing modifications? I really like your solution, but if we don't use it everywhere, people won't know what is the "right" way to do it and there's a chance they will keep using the helper functions

Reply:
Absolutely. I would much prefer to move this stuff into a modifier, but was focused on getting the ticket working first. I'd prefer to do this in the final ticket for this sub-task, after everything else is working.